eng/targets/Imports.targets

```diff
@@ -46,6 +46,15 @@
     -->
     <RoslynCheckCodeStyle Condition="'$(RoslynCheckCodeStyle)' == '' AND '$(DisableNullableWarnings)' == 'true'">false</RoslynCheckCodeStyle>
     <RoslynCheckCodeStyle Condition="'$(RoslynCheckCodeStyle)' == '' AND ('$(ContinuousIntegrationBuild)' != 'true' OR '$(RoslynEnforceCodeStyle)' == 'true')">true</RoslynCheckCodeStyle>
+
+    <!-- 
+        When markup compilation is involved, WPF generates .g.cs files via the GenerateTemporaryTargetAssembly task. 
+        These do not always have the correct full-paths to their respective .xaml files annotated in them in the #line directives.
+        This can cause compile errors if these files are included in Source Link. 
+        https://github.com/dotnet/wpf/issues/1718
```

Comment:
Believe this should be removed when we migrate to Arcade 6. How should we track removing this when that happens?

Reply:
I'm not sure why updating Arcade is related to this. It seems like a matter of fixing a WPF bug? Am I misunderstanding?

Reply:
It's because Arcade 6 has `<EmbedUntrackedSources>` set to `true` by default as well as has a work around for WPF Projects. Hence once we move to Arcade 6 this work around will be redundant with theirs

https://github.com/dotnet/arcade/blob/ca7fab569267ed3bc73360882d652d119aae5653/src/Microsoft.DotNet.Arcade.Sdk/tools/Workarounds.props#L28

Reply:
I'll just leave the linked issue open and update the description to remind us to remove this workaround.